plugins/migrate/client/cli/tx.go

```diff
@@ -0,0 +1,94 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authcmd "github.com/cosmos/cosmos-sdk/x/auth/client/cli"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+
+	"github.com/bnb-chain/node/plugins/migrate"
+)
+
+const (
+	flagBSCOperatorAddress = "bsc-operator-address"
+)
+
+func SignValidatorOwnerShipCmd(cdc *codec.Codec) *cobra.Command {
```

Comment:
This just prints the sign data. How can we get the binding relationship between the old and the new validators?